### ISSUE TITLE
I can not use pytest

### ISSUE THREAD

**Issue by joaothallis** [pytest-dev/pytest#4470]
pytest: 4.0.1
Ubuntu 18.04.1 LTS

```shell
➜ pip install -U pytest

Requirement already up-to-date: pytest in /home/joao/.asdf/installs/python/3.7.1/lib/python3.7/site-packages (4.0.1)
Requirement already satisfied, skipping upgrade: pluggy>=0.7 in /home/joao/.asdf/installs/python/3.7.1/lib/python3.7/site-packages (from pytest) (0.8.0)
Requirement already satisfied, skipping upgrade: more-itertools>=4.0.0 in /home/joao/.asdf/installs/python/3.7.1/lib/python3.7/site-packages (from pytest) (4.3.0)
Requirement already satisfied, skipping upgrade: py>=1.5.0 in /home/joao/.asdf/installs/python/3.7.1/lib/python3.7/site-packages (from pytest) (1.7.0)
Requirement already satisfied, skipping upgrade: atomicwrites>=1.0 in /home/joao/.asdf/installs/python/3.7.1/lib/python3.7/site-packages (from pytest) (1.2.1)
Requirement already satisfied, skipping upgrade: six>=1.10.0 in /home/joao/.local/lib/python3.7/site-packages (from pytest) (1.11.0)
Requirement already satisfied, skipping upgrade: attrs>=17.4.0 in /home/joao/.asdf/installs/python/3.7.1/lib/python3.7/site-packages (from pytest) (18.2.0)
Requirement already satisfied, skipping upgrade: setuptools in /home/joao/.asdf/installs/python/3.7.1/lib/python3.7/site-packages (from pytest) (39.0.1)

➜ pytest --version
zsh: command not found: pytest
```

```shell
➜ pip list
Package                  Version    Location
------------------------ ---------- --------------------------------------------------
astroid                  2.0.4
atomicwrites             1.2.1
attrs                    18.2.0
autopep8                 1.4.3
cachetools               3.0.0
certifi                  2018.10.15
chardet                  3.0.4
google-api-core          1.5.2
google-auth              1.6.1
google-cloud-core        0.28.1
google-cloud-storage     1.13.0
google-resumable-media   0.3.1
googleapis-common-protos 1.5.5
idna                     2.7
isort                    4.3.4
lazy-object-proxy        1.3.1
lxml                     4.2.5
mccabe                   0.6.1
more-itertools           4.3.0
pip                      18.1
pluggy                   0.8.0
protobuf                 3.6.1
py                       1.7.0
pyasn1                   0.4.4
pyasn1-modules           0.2.2
pycodestyle              2.4.0
pylint                   2.1.1
pytest                   4.0.1
pytz                     2018.7
requests                 2.20.1
rsa                      4.0
setuptools               39.0.1
six                      1.11.0
urllib3                  1.24.1
wrapt                    1.10.11
```

**Comment by RonnyPfannschmidt** [comment]
@joaothallis your asdf python install is set up incorrectly

**Comment by mbanerjeepalmer** [comment]
@RonnyPfannschmidt I believe I have a similar issue. I'm on Ubuntu 22.04. 

Pytest is definitely installed:
```bash
~ pip install -U pytest
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: pytest in ./.local/lib/python3.10/site-packages (7.2.0)
Requirement already satisfied: attrs>=19.2.0 in ./.local/lib/python3.10/site-packages (from pytest) (22.2.0)
Requirement already satisfied: pluggy<2.0,>=0.12 in ./.local/lib/python3.10/site-packages (from pytest) (1.0.0)
Requirement already satisfied: tomli>=1.0.0 in ./.local/lib/python3.10/site-packages (from pytest) (2.0.1)
Requirement already satisfied: iniconfig in ./.local/lib/python3.10/site-packages (from pytest) (1.1.1)
Requirement already satisfied: exceptiongroup>=1.0.0rc8 in ./.local/lib/python3.10/site-packages (from pytest) (1.1.0)
Requirement already satisfied: packaging in ./.local/lib/python3.10/site-packages (from pytest) (22.0)
```

And pip finds it:
```bash
pip show pytest
Name: pytest
Version: 7.2.0
Summary: pytest: simple powerful testing with Python
Home-page: https://docs.pytest.org/en/latest/
Author: Holger Krekel, Bruno Oliveira, Ronny Pfannschmidt, Floris Bruynooghe, Brianna Laugher, Florian Bruhin and others
Author-email: 
License: MIT
Location: /home/maurice/.local/lib/python3.10/site-packages
Requires: attrs, exceptiongroup, iniconfig, packaging, pluggy, tomli
Required-by: 
```
And its directory is definitely on my PATH
```bash
export PATH=~/.local/lib/python3.10/site-packages:$PATH
```
As a workaround I'm manually invoking:
```bash
python3 /home/maurice/.local/lib/python3.10/site-packages/pytest
```

**Comment by RonnyPfannschmidt** [comment]
Please don't comment on issues that are closed since years with entirely different problems that just seem to look the same

Also pythonpath is not the same as the path variable that's used to find executables
, please open a new issue with sufficient details to figure what you broke
